content/en/APIUseCases/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "API Use Cases"
+linkTitle: "API Use Cases"
+weight: 100
+description: >
+  Includes practical uses for our API.
+---
+
+{{% pageinfo %}}
+The pages in this book describe how you can use our API to manage your Assets
+and pentests.
+{{% /pageinfo %}}
+
+This document assumes that you can run `curl` in a command line on your system.
+You can also set up the REST calls in this book in other API clients such as:
+
+- [Postman](https://learning.postman.com/docs/getting-started/introduction/)
+- [Insomnia](https://docs.insomnia.rest/)
+
```

Comment:
## Format JSON Responses

We can make it a separate section, especially if we plan to add more sections like _Create an API Token_ and _Authorize in the API_.